shoop/order_printouts/pdf_export.py

```diff
@@ -16,10 +21,41 @@
     weasyprint = None
 
 
-def render_html_to_pdf(html):
+def _fetch_static_resource_str(resource_file):
+    resource_path = os.path.realpath(os.path.join(settings.STATIC_ROOT, resource_file))
+    if not resource_path.startswith(settings.STATIC_ROOT):
+        raise ValueError(
+            _("Possible file system traversal shenanigan detected with %(path)s") % {"path": resource_file})
```

Comment:
Don't mark exception messages as translatable, because they are only for debugging purposes and should not ever be shown to end users (and therefore translating them is wasted time). Only exception to that rule is Problem-exceptions of course, since those will be shown to end users.